src/azul/service/repository_service.py

```diff
@@ -210,8 +197,7 @@ def get_data_file(self,
                       catalog: CatalogName,
                       file_uuid: str,
                       file_version: Optional[str],
-                      filters: Filters,
-                      source_ids: Set[str]
+                      filters: MutableFilters,
```

Comment:
The docstring description of the param `filters` needs updating. I don't believe it can be an optional `None` value.